x-pack/plugins/index_management/__jest__/client_integration/home/index_templates_tab.test.ts

```diff
@@ -442,16 +441,10 @@ describe('Index Templates tab', () => {
           confirmButton!.click();
         });
 
-        const latestRequest = server.requests[server.requests.length - 1];
-
-        expect(latestRequest.method).toBe('POST');
-        expect(latestRequest.url).toBe(`${API_BASE_PATH}/delete_index_templates`);
-
-        // Commenting as I don't find a way to make it work.
```

Comment:
is this comment still valid?

Reply:
I wasnt able to make it work while I wa working on it, but just tried again and seems to be working just fine. Ill add this back :D